provider/clusterrosa/hcp/resource.go

```diff
@@ -429,6 +431,24 @@ func createHcpClusterObject(ctx context.Context,
 			return nil, errors.New(errHeadline + "\n" + errDescription)
 		}
 
+		creatorArn, ok := propertiesElements[rosa.PropertyRosaCreatorArn]
+		if !ok {
+			errDescription := fmt.Sprintf("Expected property '%s'. Please include it by supplying 'data.aws_caller_identity.current.arn'", rosa.PropertyRosaCreatorArn)
+			diags.AddError(
+				errHeadline,
+				errDescription,
+			)
+			return nil, errors.New(errHeadline + "\n" + errDescription)
+		}
+		if !rosa.UserArnRE.MatchString(creatorArn) {
+			errDescription := fmt.Sprintf("Property '%s' does not have a valid user arn. Please include it by supplying 'data.aws_caller_identity.current.arn'", rosa.PropertyRosaCreatorArn)
```

Comment:
should we use another message here for the invalid user arn, not "Please include it by supplying 'data.aws_caller_identity.current.arn'"?

Reply:
hmm, not sure what would be best. Given we are trying to have more actionable details I'd say this would be fitting. But if we want to keep ourselves a bit less opinionated we can also just say it's invalid

Reply:
@marcolan018 Adjusted so that the aws data source is treated as an example instead. Let me know if this is better